lib/purerlex.ex

```diff
@@ -54,7 +54,14 @@ defmodule Mix.Tasks.Compile.Purerl do
 
     cached = read_cache(project_root_probably)
 
-    files = Enum.flat_map(@file_paths, &Path.wildcard/1)
+    purs_files = Enum.flat_map(@file_paths, &Path.wildcard/1)
+
+    erl_files =
+      purs_files
+      |> Stream.map(&String.replace(&1, ~r/\.purs\z/, ".erl"))
```

Comment:
Use https://hexdocs.pm/elixir/1.12/String.html#replace_suffix/3 instead of regex

Reply:
Fixed in https://github.com/drathier/purerlex/pull/1/commits/0a791e20f219bd3b58d31b53e1dff7fe287c1c27